web/context.go

```diff
@@ -85,14 +85,33 @@ func (c *Context) LogAuditWithUserId(userId, extraInfo string) {
 	}
 }
 
+func (c *Context) LogByCode(err *model.AppError) {
```

Comment:
Let's rename it to `LogErrorByCode` to make it clear what does it log.

Additionally, I think we should refactor out the actual log line printing from all the 3 methods to a separate internal method. This should keep all the fields that get printed in a single place. That method should only take the level and the apperror as its arguments.

Reply:
Updated, in fact removed all other 3 methods and now we can have one method for all cases.